README.md

```diff
@@ -1,18 +1,110 @@
 # ember-navigator
 
-**Note: Very Very Alpha**
+**Note: Very Alpha. Under-documented. Intended only for experimenters and intrepid early adopters.**
 
-The current focus is landing an Ember-equivalent API of
-[React Navigation](https://reactnavigation.org/) library, which is a
-robust, battle-tested, and well designed navigation solution.
+This project aims to provide an alternative to the Ember Router that is better-suited to mobile navigation patterns.
+In particular, many mobile apps have a things like a tab bar where one tab is active at a time, and each tab has a
+stack-style navigation where screens are pushed onto the stack as a user drills down into content and the back button
+pops screens off the stack. The order, quantity, and types of screens pushed onto a stack can vary considerably, which
+makes the built-in Ember router poorly suited for this use case.
+
+It is heavily influenced by the API of the [React Navigation](https://reactnavigation.org/) library, which is a
+robust, battle-tested, and well-designed navigation solution for React Native.
 
 ## Installation
 
-* `git clone <repository-url>` this repository
-* `cd ember-navigator`
-* `yarn install`
+`ember install ember-navigator`
+
+## Concepts
+
+### Router "map"
+
+The router map should seem familiar from ember's router, with some differences. Here's an example of what the router
+map might looks like for an app similar to Twitter:
+
+```js
+tabRouter('tabs', [
```

Comment:
How about doing a full example if the file with imports and all?

Reply:
Done. Thanks for reviewing. Let me know what else could be clearer or less confusing, and what questions you have after reviewing.